src/Composer/ComposerizeDrupalCommand.php

```diff
@@ -360,6 +366,18 @@ protected function findContribProjects($root_composer_json)
             $root_composer_json
         );
         $projects = array_merge($modules, $themes, $profiles);
+        $contrib_dirs = $this->input->getArgument('contrib-dir');
+        if ($contrib_dirs) {
+            foreach ($contrib_dirs as $dir) {
+                $contrib_projects =  DrupalInspector::findContribProjects(
+                    $this->drupalRoot,
+                    $dir,
+                    $root_composer_json
+                );
+                $projects = array_merge($contrib_projects, $projects);
```

Comment:
This may cause duplicates, especially if someone passes a value like `modules/contrib`.